addon/components/variable-number/number.ts

```diff
@@ -35,6 +36,7 @@ export default class VariableNumberPluginNumberComponent extends Component<Args>
   @tracked errorMessage = '';
   @service declare intl: intlService;
   cursorPositionKeyDown: number | null = null;
+  @tracked infoTooltipOpen = false;
```

Comment:
am I missing something, or is this unused code that should be removed?

Reply:
Yes it should be removed indeed :)